src-bindings/node/node.ml

```diff
@@ -44,7 +44,7 @@ module Process = struct
 
     let set k v = Ojs.set_prop_ascii env k ([%js.of: string] v)
 
-    include [%js: val env : string Interop.Dict.t [@@js.global "process.env"]]
```

Comment:
Can't we just use this?

https://github.com/ocamllabs/vscode-ocaml-platform/blob/0bbcdb36ea6fe8faa473eca5c5efe219af462dab/src-bindings/node/node.ml#L41

Reply:
Never mind!